RichClipboard.package/RCClippingsList.class/instance/initialize.st

```diff
@@ -2,11 +2,15 @@ initialize-release
 initialize
 	super initialize.
 	self borderWidth: 0.
-
 	regexFilter := nil.
 	self contextMenuIsOpen: false.
+	self timeoutMS: self class defaultTimeoutMS.
 	
 	self storage
 		when: #clippingsChanged
 		send: #clippingsChanged 
+		to: self.
+		
+	systemClipboard when: #contentChanged
+		send: #deleteSystemClipboardTextAfterTimeout
 		to: self
```

Comment:
Das Aufräumen nicht vergessen! Siehe RichClipboard>>destroy.

Allgemein: Warum habt ihr die gesamte Funktionalität in RCClippingsList implementiert? Ich finde das hat hier nichts verloren. Eigentlich muss das meiner Meinung nach direkt in "RichClipboard" und dort könnt ihr dann auch den bestehenden Eventlistener für #contentChanged benutzen.
Wenn das in dieser Klasse ist besteht nämlich auch das Problem, dass es einmal für jedes Fenster ausgeführt wird, was natürlich totaler Quatsch ist.

Ob ihr euch im privateMode befindet, könnt ihr in RichClipboard über "storage privateModeActive" überprüfen.

Reply:
Ich sehe grade, dass ihr gar keine Tests habt. Von daher ist diese Variable ja sowieso unnötig